packages/web/src/pages/sign-up-page/pages/FinishProfilePage.tsx

```diff
@@ -112,9 +112,9 @@ export const FinishProfilePage = () => {
   const handleSubmit = useCallback(
     ({ coverPhoto, profileImage, displayName }: FinishProfileValues) => {
       dispatch(setValueField('name', displayName))
-      dispatch(setField('profileImage', { value: profileImage }))
+      dispatch(setField('profileImage', profileImage))
```

Comment:
nice so all-along we shouldn't have been using `{value: ...}`, ie old sign-up doesn't do this?

Reply:
yea the saga wasn't looking in the value prop so i just removed it. I also dont think that the editablefield props like status and error were being used